tidy3d/__init__.py

```diff
@@ -78,6 +78,7 @@
 # material library dict imported as `from tidy3d import material_library`
 # get material `mat` and variant `var` as `material_library[mat][var]`
 from .material_library.material_library import material_library
+from .material_library.parametric_materials import Graphene
```

Comment:
ok, if I put it in the material_library, then the syntax is `material_library["graphene"]()`

Reply:
That's similar to how we define the other material library items right? How about we leave Graphene in `__init__.py` just because it might be used so often but also generally include parametric materials in the material library.

Reply:
Reason being that someone might want to look at all the materials in the library and wonder where graphene is or whether we support it, they might not think to look in the `tidy3d` namespace.